src/Model/Group.php

```diff
@@ -148,12 +148,14 @@ public static function getIdsByContactId(int $cid): array
 
 		// Meta-groups
 		$contact = Contact::getById($cid, ['rel']);
-		if ($contact['rel'] == Contact::FOLLOWER || $contact['rel'] == Contact::FRIEND) {
-			$return[] = self::FOLLOWERS;
-		}
+		if ($contact) {
```

Comment:
Wouldn't it be better to return here when the contact wasn't found instead of encapsulating the following statement?

Reply:
Not necessarily since we don't throw an Exception or return anything different from `$return`. However, I can rewrite this so that we first perform the contact retrieval and return early without fetching the group appartenance. It may even fix an issue where contacts still show in groups even though they don't in the contact list.

Reply:
What I meant was, that I like to prevent nesting. So when we can quit a function early, then I think we should do it, since this avoids the need for some
```
if ($foo) {
...
...
...
}

Reply:
I get it, but I don't see the difference between

```
if (foo) {
   ...
   ...
   ...
}

return $return;
```
and
```
if (!foo) {
    return $return;
}

...
...
...

return $return;
```
If anything the second version is more confusing because `$return` gets returned twice with no functional difference.

Reply:
Anyway, it's rewritten now.